x-pack/plugins/cases/server/saved_object_types/migrations/comments.ts

```diff
@@ -175,3 +184,29 @@ export const mergeMigrationFunctionMaps = (
 
   return mergeWith({ ...obj1 }, obj2, customizer);
 };
+
+export const removeRuleInformation = (
```

Comment:
Do we need to do the same for user actions?

Reply:
oooh great point. I'll add that tomorrow!